examples/live/run.sh

```diff
@@ -47,7 +70,8 @@ function simulate_strain { # detector PSD_model random_seed
         --sample-rate 16384 \
         --low-frequency-cutoff 10 \
         --channel-name $1:SIMULATED_STRAIN \
-        --frame-duration 32
+        --frame-duration 32 \
+        --injection-file hwinjcbc_1272790426.xml
```

Comment:
I would prefer not having to hardcode the XML file name, but I see `pycbc_generate_hwinj` is not super flexible in controlling the name, so I think this is ok for now.